shardingsphere-sql-parser/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/BaseRule.g4

```diff
@@ -110,9 +110,13 @@ unreservedWord
     ;
 
 variable
-    : (AT_? AT_)? (GLOBAL | PERSIST | PERSIST_ONLY | SESSION)? DOT_? identifier
+    : (AT_? AT_)? scopeKeyword? DOT_? identifier
```

Comment:
Please consider about how to name to rule, other rules never contain like appendix of `keyWord`

Reply:
done